.rubocop.yml

```diff
@@ -14,10 +14,14 @@ AllCops:
 
 Rails/HasManyOrHasOneDependent:
   Enabled: false
+Rails/InverseOf:
```

Comment:
How comes we are overriding this?

Reply:
This is because of the `has_one` I added on `Course` as a small optimisation compared to the previous version which fetches all the enrichments and does the find in Ruby. Because there are conditions on it there's not really an inverse association for this and is kind of a one way thing